doc/sphinx/src/outputs.rst

```diff
@@ -158,6 +161,134 @@ This will produce a text file (``.hst``) output file every 1 units of
 simulation time. The content of the file is determined by the functions
 enrolled by a specific package, see :ref:`state history output`.
 
+Histograms
+----------
+
+Parthenon supports calculating flexible 1D and 2D histograms in-situ that
+are written to disk in HDF5 format.
+Currently supported are
+
+- 1D and 2D histograms
+- binning by variable or coordinate (x1, x2, x3 and radial distance)
+- counting samples and or summing a variable  
+- weighting by volume and/or variable
+
+The output format follows ``numpy`` convention, so that plotting data
```

Comment:
Does this mean it follows C-ordering for 2D arrays?

Reply:
yes, so the data output data uses `[x,y]` indexing with `y` being the fast index. I'll update the doc do be more specific here, too.